sonnet/src/bias.py

```diff
@@ -125,8 +125,7 @@ def __call__(self, inputs: tf.Tensor, multiplier: types.FloatLike = None):
     self._initialize(inputs)
     if multiplier is not None:
```

Comment:
So it seems to me we have a pattern of :

```python
if A:
  return B + (C * D)
else:
  return B + D
```

I think perhaps in this case, if we want to avoid the extra else perhaps we should extract more of the common part of the two expressions (e.g. the `if A` block should just scale the thing we add to `B`.

```python
if A:
  D = C * D
return B + D
```

Concretely in this case it would look like the following (I think we cannot use `*=` because this is not supported on `tf.Variable`):

```python
b = self.b
if multiplier is not None:
  b = b * multiplier
return inputs + b
```

Reply:
I agree. Thanks You!

Reply:
Sounds good, please feel free to push a new commit with these changes so we can review and get them merged 😄